src/cocotb/handle.py

```diff
@@ -1104,11 +1285,10 @@ def SimHandle(
     """Factory function to create the correct type of `SimHandle` object.
```

Comment:
Maybe because `SimHandleBase` is both a return type annotation and a reference in the "Returns:" part of the docstring?
It's not duplicated in the old version, https://docs.cocotb.org/en/latest/library_reference.html#cocotb.handle.SimHandle

Reply:
I ran into this in other cases as well, like on the `value` getters and setters. It's a bug in Sphinx that has something to do with the cache AFAICT. I rebuild fixed the errors that I saw locally.

Reply:
Looks like still an issue but I'm fine with merging and seeing if it is fixed on master.

Reply:
Yeah there's something up with the cache. I fixed the missing `"` character from your last comment, but it's still missing from the documentation. Maybe master will fix itself. I wonder if there's a way to clear or not use the cache that we can set in `.readthedocs.yml`